src/config/wmodules-command.c

```diff
@@ -58,12 +61,25 @@ int wm_command_read(xml_node **nodes, wmodule *module, int agent_cfg)
             }
         } else if (!strcmp(nodes[i]->element, XML_TAG)) {
             if (strlen(nodes[i]->content) == 0) {
-                merror("Empty content for tag '%s' at module '%s'.", XML_TAG, WM_COMMAND_CONTEXT.name);
-                return OS_INVALID;
+                mwarn("Empty content for tag '%s' at module '%s'.", XML_TAG, WM_COMMAND_CONTEXT.name);
+                command_tag_length = strlen(WM_COMMAND_CONTEXT.name) + 2;
+                command_tag = malloc(sizeof(char) * command_tag_length);
+                snprintf(command_tag, command_tag_length, "%s", WM_COMMAND_CONTEXT.name);
+                os_strdup(command_tag, nodes[i]->content);
```

Comment:
`os_strdup` sets from left to right. In this case, it would be overwriting the content of the XML structure and producing a memory leak.

Did you mean...?
```c
os_strdup(command_tag, command->tag);
```